client/internal/dns/server.go

```diff
@@ -219,41 +221,50 @@ func (s *DefaultServer) UpdateDNSServer(serial uint64, update nbdns.Config) erro
 			s.updateSerial = serial
```

Comment:
I found a bug with the hash configuration, where a nameserver change without replacing the number of IPs caused the hashes to be identical. Can you add `UseStringer:     true,` to `hashstructure.HashOptions` in line 210?

Reply:
Fix added